server/text_generation_server/models/causal_lm.py

```diff
@@ -608,6 +608,7 @@ def forward(
     ) -> Tuple[torch.Tensor, List[Tuple[torch.Tensor, torch.Tensor]], int]:
         model_inputs = self.model.prepare_inputs_for_generation(
             input_ids, past_key_values,
+            inputs_embeds=inputs_embeds,
```

Comment:
I'm curious about this change. What changed in transformers that makes us need this now (though I see that we should have it now anyways)?